src/examples/button/SplitButtonTypes.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { SplitButton, Button, ChevronButton } from '@zendeskgarden/react-buttons';
+import { Dropdown, Trigger, Menu, Item } from '@zendeskgarden/react-dropdowns';
+
+const StyledCol = styled(Col)`
+  @media (max-width: ${p => p.theme.breakpoints.sm}) {
+    margin-top: ${p => p.theme.space.sm};
+  }
+`;
```

Comment:
Technically, from what I'm seeing, this example doesn't need a responsive layout (in combo with the `sm` breakpoints below). The groups continue to layout side-by-side on small screens. Compare with https://zendeskgarden.netlify.app/components/button#types. But this is just a nit for example simplification. Fine to leave as-is if preferred.

Reply:
TY, fixed